projects/plugins/jetpack/tests/e2e/config/encrypted.enc

```diff
@@ -1,2 +1 @@
-Salted__BHø��߂�U�j�/���X�П_�T3��'���Ax��M{(��
-�J��"Fr��A%�&+���G�M�Nj�|m�F����P�
\ No newline at end of file
+Salted__dR!(d�$+U�Qfx%���ͨ�We�f��f!q1L��Ldkk��z��g�X��m�~�#��7��K&?RW��ƙ��h��8���
```

Comment:
I can't seem to get this to work. When I run `pnpm config:decrypt`, the generated `projects/plugins/jetpack/tests/e2e/config/local.cjs` still includes a `require( 'jetpack-e2e-commons/` reference. Does it work for you?

Reply:
Works for me. Note it changes `require( 'jetpack-e2e-commons/` to `require( '_jetpack-e2e-commons/`, just the one character change.

Also the E2E tests themselves seem happy with it.

Reply:
I tested again, and now it works. I hadn't seen that the decryption had failed yesterday, so I was still seeing an old decrypted file.